tools/sistr_cmd/sistr_cmd.xml

```diff
@@ -0,0 +1,227 @@
+<tool id="sistr_cmd" name="sistr_cmd" version="0.3.1">
+  <description>
+    Salmonella In Silico Typing Resource commandline tool for serovar prediction
+  </description>
+  <requirements>
+    <requirement type="package" version="0.3.1">sistr_cmd</requirement>
+  </requirements>
+  <stdio>
+    <exit_code range="1:" />
+  </stdio>
+  <command><![CDATA[
+  sistr 
+    -f $output_format
+    #if $output_format == "csv"
+      -o sistr-report.csv
+    #elif $output_format == "json"
+      -o sistr-report.json
+    #end if
+    -p $cgmlst_profiles
+    -n $novel_alleles
+    -a $alleles_output
+    -T $tmp_dir
+    #if $keep_tmp
+      -K
+    #end if
+    #if $use_full_cgmlst_db
+      --use-full-cgmlst-db
+    #end if
+    #if $no_cgmlst
+      --no-cgmlst
+    #end if
+    #if $run_mash
+      -m
+    #end if
+    #if $qc
+      --qc
+    #end if
+    #if $threads > 1
+      -t $threads
+    #end if
+    #if $verbosity == 1
+      -v
+    #elif $verbosity == 2
+      -vv
+    #elif $verbosity == 3
+      -vvv
+    #end if
+    ${" ".join(map(str, $input_fastas))}
+  ]]></command>
+  <inputs>
+    <param 
+      name="input_fastas" 
+      type="data" 
+      label="Input Genome(s)" 
+      optional="false" 
+      multiple="true"
+      format="fasta"
+      />
+    <param 
+      name="output_format" 
+      type="select" 
+      label="Results output format"
+      multiple="false" >
+      <option value="csv" selected="true">
+        CSV (Comma Separated Values)
+      </option>
+      <option value="json">
+        JSON (JavaScript Object Notation)
+      </option>
+    </param>
+    <param 
+      name="use_full_cgmlst_db"
+      type="boolean"
+      checked="false"
+      label="Use full cgMLST database for serovar prediction. About 10X slower with equivalent results to reduced centroid allele database."
+      />
+    <param 
+      name="run_mash"
+      type="boolean"
+      checked="true"
+      label="Run Mash MinHash-based serovar prediction"
+      />
+    <param 
+      name="no_cgmlst"
+      type="boolean"
+      checked="false"
+      label="Skip running cgMLST-based serovar prediction"
+      />
+    <param 
+      name="qc"
+      type="boolean"
+      checked="true"
+      label="Basic QC of results"
+      />
+    <param
```

Comment:
Instead of allowing a user of Galaxy to set the temporary directory, could you use the `$TMPDIR` environment variable instead?  And default to `/tmp`?  This way, a user of Galaxy could never start writing random files any place the Galaxy user has access to.  You should be able to use the syntax `"\${TMPDIR:-/tmp}"`, similar to `GALAXY_SLOTS` above.